tests/testsuite/lints/unused_optional_dependencies.rs

```diff
@@ -221,7 +221,7 @@ Caused by:
 error: failed to parse manifest at `[ROOT]/foo/Cargo.toml`
 
 Caused by:
-  feature `feat` includes `dep?/feat`, but `dep` is not a dependency
+  feature `feat` includes `dep?/feat`, activate it in a feature with `dep:dep`
```

Comment:
This never finishing saying whats wrong but jumps straight to a suggestion.

e.g.
```
feature `feat` includes `dep?/feat` but no feature activates `dep`.
To activate `dep`, a feature must specify `dep:dep`.
```
```
`dep?/feat` (in `feat`) activates `feat` when `dep` is activated but no feature activates `dep`.
To activate `dep`, a feature must specify `dep:dep`.
```
```
feature `feat` includes `dep?/feat` but no feature activates `dep` with `dep:dep`.
```

Reply:
Oops， my local commit dosn't push to here, but it is similar to this
> feature `feat` includes `dep?/feat` but no feature activates `dep` with `dep:dep`.